### ISSUE TITLE
Review how we use numbers

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual-backlog#52]
There's been a suggestion from the live service content team that we review how we use numbers. 

Here is what we say in the content style guide at the moment: https://beta.nhs.uk/service-manual/content/formatting-and-punctuation

And one suggestion is that we follow the readability guidelines at:http://readabilityguidelines.wikidot.com/numbers

There was a proposal on Slack that we make this clearer. See https://nhsuk.slack.com/archives/C644HFZ9N/p1553528638190600

Added as a task to Jira:  Jira SS-579

**Comment by sarawilcox** [comment]
Agreed to change our current wording in the style guide. A few issues to resolve:
- ordinal numbers - check Readability guidelines and any issues with assistive tech https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/100
- check truncated dates on assistive tech https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/99

**Comment by sarawilcox** [comment]
All issues resolved at 18 July Style Council meeting. Page updated to reflect changes.

**Comment by sarawilcox** [comment]
A number of content designers have reported problems with our latest guidance on using 1 and 2 where one would work better. 
https://beta.nhs.uk/service-manual/content/numbers-measurements-dates-time
We're collecting examples and will review at December Style Council meeting.

**Comment by sarawilcox** [comment]
![Screenshot_20191125-223348](https://user-images.githubusercontent.com/45792959/69625415-6ebb3200-103e-11ea-80e1-1824b86635d5.png)
A colleague thought the 1 here was a typo.

**Comment by sarawilcox** [comment]
Another example: "You can use a nebuliser in hospital or you may be given one to manage your condition at home." (Here 1 might suggest that you would only be given 1.)

**Comment by JackMatthams** [comment]
- "Never take 2 doses at the same time to make up for a forgotten **one**." Codeine: Medicines A to Z. I think this is correct. 
- "It can help to use post-its, or print out and cut up a paper prototype, to see how the answers might take users on **1** journey or another." Use filter questions to route users: Service manual. This doesn't feel numerical, so should be "one"?
- "Several short questions are often easier than **1** long **one**." Think of form as a conversation: Service manual. A good example of the difference. 
- "the consequences of making **1** choice over another" Write the supporting content for your form: Service manual. Again, this doesn't seem numerical. 
- "If you begin prototyping with 1 thing per page, you can deal with errors for 1 thing at a time."

**Comment by sarawilcox** [comment]
I think we've covered this now in #162 and #197. @JackMatthams, that's right, isn't it? If so, we can push this to Done.

**Comment by sarawilcox** [comment]
We're keeping this open as the main Numbers ticket. The numbers page in the style guide is now up to date: https://service-manual.nhs.uk/content/numbers-measurements-dates-time

I'm closing down these superseded/done numbers tickets:
- [Measurements: Change to numbers page and other page](https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/134)
- [Numbers page - minor changes](https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/133)
- [Ordinal numbers](https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/100)
